src/Playwright.Tests/ClientCertficatesTests.cs

```diff
@@ -118,7 +118,6 @@ public async Task ShouldWorkWithNewContext()
     {
         var context = await Browser.NewContextAsync(new()
         {
-            // TODO: Remove this once we can pass a custom CA.
```

Comment:
drive-by: remove TODO comments as per https://github.com/microsoft/playwright/pull/31973 - since we don't plan to follow-up there.